maps/includes/base_shutdown.lua

```diff
@@ -88,6 +88,8 @@ function button_common:allowed( allowed_entity )
 	if IsPlayer( allowed_entity ) then
 		local player = CastToPlayer( allowed_entity )
 		if player:GetTeamId() == self.team and player:IsAlive() then
+			player:AddFortPoints(100, "disabled security")

```

Comment:
The 100 (or whatever number of points it ends up being) should also be abstracted out into a global variable (`FORTPOINTS_PER_SECURITY_DISABLE` for example).